src/components/molecules/NavbarRadio/NavbarRadio.scss

```diff
@@ -6,5 +6,15 @@
     position: absolute;
     top: 100%;
     right: 0;
+
+    &--visible {
+      visibility: visible;
+      user-select: auto;
+    }
+
+    &--hidden {
+      visibility: hidden;
+      user-select: none;
+    }
```

Comment:
In cases where our variants are 'boolean' (aka: one or the other), and we need one of them to exist to be 'valid', then I tend to choose one and make it the default case (in the base class), and then the 'non-standard case' becomes the variant.

Looking at the component code, this seems to be used to show/hide the radio popover, so I would say the default case is likely going to be hidden:

(also thinking something like `--show` or `--show-drawer` or similar might be better naming, for this pattern of 'thing that slides out from the NavBar')

```suggestion

    visibility: hidden;
    user-select: none;

    &--show {
      visibility: visible;
      user-select: auto;
    }
```

---

Generally speaking I would also tend to set the variants on the 'main class' rather than one of the `__elements` (unless there's a particularly strong reason why it needs to be set on the `__element`). Even when it's overriding styles on the element, i'll structure it something like this:

```scss
.FooMainThing {
  &--bar {
    .FooMainThing__thing {
      // some style overrides
    }
  }

  &__thing {
    // some default styles
  }
}
```

---

[NB] Do we need to set `user-select` when `visibility: hidden`? I would have guessed it would make no difference? What about `display: none`, is that a good/better choice than `visibility` here? And if so/not, why/why not?

- https://developer.mozilla.org/en-US/docs/Web/CSS/user-select
- https://developer.mozilla.org/en-US/docs/Web/CSS/visibility
  - > `hidden`: The element box is invisible (not drawn), but still affects layout as normal. Descendants of the element will be visible if they have `visibility` set to `visible`. The element cannot receive focus (such as when navigating through tab indexes).
- https://developer.mozilla.org/en-US/docs/Web/CSS/display

Reply:
> Honestly, 95% of these have been copy pasted from @mcflyok's design.

Which is fine, but that doesn't make them 100% perfect or accurate or to be blindly followed. They're a guide to be combined with our engineering knowledge and practices.

I'm not sure what your issue/concern is here...? I'm asking questions as part of code review as is our standard/the purpose of code review..?

(cc // @jonboutelle Perhaps you can dig into this and find out what the deeper underlying concerns/issues are, as I feel like I am struggling to)